base_location/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Location management (aka Better ZIP)',
-    'version': '12.0.1.0.1',
+    'version': '12.0.1.0.2',
```

Comment:
you are not doing any migration, why increment the version?

Reply:
The version bump may happen several times:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers